skstore/ts/src/prv/skstore_impl.ts

```diff
@@ -239,6 +241,164 @@ class EHandleImpl<K extends TJSON, V extends TJSON> implements EHandle<K, V> {
     return this.mapN(mapper, p1, p2, p3, p4, p5, p6, p7, p8, p9);
   }
 
+  mapValuesN<
+    V2 extends TJSON,
```

Comment:
I wonder what the feeling is about using `V` and `V2` as opposed to e.g. `V` and `W`? My impression is that, especially when speaking out loud, it is easier to have either all variables like this be numbered, or to use different letters, but mixing is harder. When there are only a few, my feeling is that using different letters is easier.

(I realize this is not specific to this PR, and feel free to ignore, but it seems like a reasonable point to discuss the question.)

Reply:
I don't have strong feelings about this so switched from V2 to W throughout!  I agree that mixing numbered and non-numbered type variables gets a bit confusing.